### PR TITLE
Create Hotel Prices.cpp

### PR DESCRIPTION
**Pull request by Harshit-Poddar90**
### Issue Id you have worked upon - 
#244 

### Briefly explain your program logic - 
-use different types of data types and loops.
### Screenshots(Attach 2 screenshots of your own input and output) - 
image

<br>

### Checklist:
Eg - If your code follow the below guidelines. Kindly change [] to [x]

All the conditions should be fulfilled for considering your code for merging -

[x] I have mentioned the question as comment in my solution file.<br>
[x] My code follows the guidelines of this project.<br>
[x] I have performed a self-review of my own code.<br>
[x] I have commented my code.<br>
[x] My code gives the correct output.<br>
[x] I confirm that I have not copied the code from anywhere. In case its found that I have copied I can be banned from the repository.<br>

[x] I affirm that I strictly follow contributing guidelines and code of conduct.

### CODE REVIEW

**Comment by SarthakKeshari** [comment]
@Harshit-Poddar90,
Kindly add comments to your solution file.

**Comment by Harshit-Poddar90** [comment]
ok

File: Hackerrank/Hotel Prices/Hotel Prices.cpp [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+        int bedrooms_;
+        int bathrooms_;
+};
+
+class HotelApartment : public HotelRoom {
+public:
+    HotelApartment(int bedrooms, int bathrooms) 
+    : HotelRoom(bedrooms, bathrooms) {}
+
```
Comment (Harshit-Poddar90) [inline Hackerrank/Hotel Prices/Hotel Prices.cpp:48]: //price of hotelrooms

**Comment by Harshit-Poddar90** [comment]
I have added some basic comments and I have also put the questions.

**Review by SarthakKeshari — APPROVED** [review]

**Pull request merged**